package.json

```diff
@@ -42,5 +42,5 @@
     "Ognian Tschakalov <ognian.tschakalov@ogi-it.com>",
     "Simon Wade <simon.wade@gmail.com>"
   ],
-  "license": "BSD"
+  "license": "BSD-3-Clause"
```

Comment:
The BSD license is the 4-Clause license: https://en.wikipedia.org/wiki/BSD_licenses#4-clause_license_.28original_.22BSD_License.22.29

So we can't just change the license like that.  We have to contact @villadora to do that (maybe we can change it to another license right away). I'll accept the change with a `BSD-4-Clause` definition.